src/skani.rs

```diff
@@ -78,14 +80,8 @@ fn precluster_skani(
                 let query_name = sketches[j].file_name.clone();
```

Comment:
@wwood I tried using `String::from(ref_sketch.file_name)` and  `String::from(ref_sketch.file_name.clone())` but get the same error. Did you mean somewhere else?